src/js/controller.js

```diff
@@ -217,7 +217,7 @@ searchContainer.addEventListener('submit', async e => {
   e.preventDefault();
   spinner(searchResults);
   await searchDataResult(searchBox.value);
-  console.log(e, 'sdf', state.search.results);
+  // console.log(e, 'sdf', state.search.results);
```

Comment:
commented logs are not needed in production, you should have that only while debugging somethin